src/services/navigationBar.ts

```diff
@@ -192,6 +193,11 @@ namespace ts.NavigationBar {
                             addTopLevelNodes((<Block>functionDeclaration.body).statements, topLevelNodes);
                         }
                         break;
+
+                    default:
+                        const childrens: Node[] = [];
+                        forEachChild(node, child => { childrens.push(child) });

```

Comment:
i woudl split this into two functions, addTopLevelNodes:
```ts
        function addTopLevelNodes(nodes: Node[], topLevelNodes: Node[]): void {
            forEach(nodes, addTopLevelNode);
        }
```

Now the next issue is to figure out what to do with the sorting. i believe we can sort the output instead.